server/src/compression.rs

```diff
@@ -398,14 +398,20 @@ mod tests {
     use super::*;
 
     use datafusion::arrow::array::UInt8Array;
+    use rand::{thread_rng, Rng};
 
+    use crate::compression::test_util as comp_test_util;
     use crate::metadata::test_util;
     use crate::models;
 
+    const ERROR_BOUND_ZERO: f32 = 0.0;
+    const ERROR_BOUND_FIVE: f32 = 5.0;
+    const TRY_COMPRESS_TEST_LENGTH: usize = 50;
+
     // Tests for try_compress().
```

Comment:
Sorry I did not think of it in the first round of reviews, but since the tests for `try_compress()` are almost identical, maybe the amount of duplicate code can be reduced by creating a function with the shared code.

Reply:
Fixed.